ext/fileinfo/libmagic/apprentice.c

```diff
@@ -1542,7 +1542,7 @@ parse(struct magic_set *ms, struct magic_entry *me, const char *line,
 				file_oomem(ms, sizeof(*nm) * cnt);
 				return -1;
 			}
-			me->mp = m = nm;
+			me->mp = nm;
```

Comment:
These probably need to be submitted upstream to fileinfo, we try to avoid going too far from the upstream.